addons/digest/data/digest_data.xml

```diff
@@ -107,11 +109,16 @@
             <td align="center" valign="top">
                 <table bgcolor="#ffffff" cellspacing="0" cellpadding="0" width="650" align="center" border="0" style="border: 1px solid #eeeeee; border-bottom: none;width: 100%; max-width: 650px; padding:30px 30px">
                     <tr t-foreach="kpi_data" t-as="kpi_info">
-                        <td>
+                        <td t-if="kpi_info.get('kpi_col1') or kpi_info.get('kpi_col2') or kpi_info.get('kpi_col3')">
                             <table cellspacing="0" cellpadding="0" border="0" width="580" align="center" style="width:100%; max-width:580px;">
                                 <tr>
                                     <td align="left" style="border-bottom: 1px solid #eeeeee;">
-                                        <span style="color:#282f33;font-size: 15px; font-weight: bold; line-height: 30px" t-esc="kpi_info['kpi_fullname']"/>
+                                        <t t-if="kpi_info['kpi_fullname_raw']">
+                                            <t t-raw="kpi_info['kpi_fullname']"/>
```

Comment:
Same here, can't we have a way to avoid t-raw and let html in digest template ? It's mostly simple styling so I bet with some parameters it id doable.

Of course digest template becomes harder to use and understand but I think it is better to avoid t-raw, even for technical templates.

Reply:
@tde-banana-odoo For this one I'm not sure it's a good idea, the title of the section in the KPI mail is very specific...

**Engagement** on **_3000_** Emails sent

(that's why I build the title outside of the template)

Reply:
Is it mandatory to have such strange formatting ? We could negociate a full-bold title. I think current solution is more word-art 2003 than current html design ...

Reply:
@tde-banana-odoo So we just display simple text like this ?
![_tmp_screenshot](https://user-images.githubusercontent.com/54316226/84488841-14638500-aca1-11ea-910a-ec9f52e5937f.png)

Reply:
We have to convince lna but for me it is better (simpler formatting, simpler template :D) a different styling each word is quite ugly in my opinion

Reply:
LNA agrees on discord :D